src/calendar/view/CalendarView.ts

```diff
@@ -216,6 +232,7 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 									startOfTheWeek: downcast(locator.logins.getUserController().userSettingsGroupRoot.startOfTheWeek),
 									dragHandlerCallbacks: this.viewModel,
 									isDaySelectorExpanded: this.viewModel.isDaySelectorExpanded(),
+									weekIndicator: this.navConfiguration.week,
```

Comment:
I think the only reason to keep weekIndicator in `NavConfiguration` was to share it across the toolbars. We can still pass it in pre-computed but it doesn't have to be in nav config anymore I think